### PR TITLE
Add getTypeDir in upload_image_insert_db

### PR DESCRIPTION
**Pull request by stvreumi**

### CODE REVIEW

**Comment by stvreumi** [comment]
I separate that just because I think that line maybe too long.
I finish the modification.

File: server_api.py [hunk 1]
```diff
@@ -677,10 +677,10 @@ def upload_image_insert_db(json_obj):
             return_msg['result'] = receive_msg['error']
         
         #get new file place
-        sql = ("SELECT type_dir FROM data_type WHERE type_id=" + str(type_id))
-        pure_result = db.query(sql)
+        with DataTypeDao() as dataTypeDao:
+            result = dataTypeDao.getTypeDir(typeId=str(type_id))
```
Comment (Billy4195) [inline server_api.py]: I think `type_dir` is better than `result`

File: server_api.py [hunk 2]
```diff
@@ -677,10 +677,10 @@ def upload_image_insert_db(json_obj):
...
         try:
-            img_new_file_dir = os.path.join(server_dir, "static/"+str(pure_result[0][0]))
+            img_new_file_dir = os.path.join(server_dir, "static/{dir_name}".format(dir_name=result))
```
Comment (Billy4195) [inline server_api.py]: Same as above, `type_dir` is better than `dir_name`

**Comment by stvreumi** [comment]
Finish modification.

**Pull request merged**